pkg/uconn/analyzer.go

```diff
@@ -145,8 +145,6 @@ func (a *analyzer) hostMaxDurQuery(maxDur float64, localIP data.UniqueIP, extern
 	// create query
 	query := bson.M{}
 
-	//TODO: Why do we update old chunks with new max durations??? (https://github.com/activecm/rita/pull/512/files) -LL
```

Comment:
While this code could be refactored, generally, we create a new entry for each chunk and update all of the entries if an entry already exists for the current chunk.